READMEs/get-rose-on-sapphire.md

```diff
@@ -0,0 +1,22 @@
+# Get ROSE tokens on Sapphire Mainnet
+
+## Get ROSE tokens on Oasis Emerald Network
```

Comment:
Above, I give tips for "get-ocean-on-sapphire". Please apply those tips to this README too.